README.md

```diff
@@ -58,42 +95,235 @@ import ReactDOM from 'react-dom';
   }, []);
 
+  function onAdd(name) {
+    add({ name });
+  }
+
+  function onRemove(index) {
+    remove(index);
+  }
+
+  function onCheck(index) {
+    toggle(index);
+  }
+
   return (
     <div>
-      <h2>Todo</h2>
-      <ul>
-        {dataSource.map(({ name }) => (
-          <li>{name}</li>
-        ))}
-      </ul>
+      <h2>Todos</h2>
+      {!refresh.loading ? (
+        dataSource.length ? (
+          <ul>
+            {dataSource.map(({ name, done }, index) => (
+              <li key={index}>
+                <label>
+                  <input
+                    type="checkbox"
+                    checked={done}
+                    onClick={() => onCheck(index)}
+                  />
+                  {done ? <s>{name}</s> : <span>{name}</span>}
+                </label>
+                <button onClick={() => onRemove(index)}>-</button>
+              </li>
+            ))}
+          </ul>
+        ) : (
+          "no task"
+        )
+      ) : (
+        "loading..."
+      )}
+      <div>
+        <input
+          onKeyDown={event => {
+            if (event.keyCode === 13) {
+              onAdd(event.target.value);
+              event.target.value = "";
+            }
+          }}
+          placeholder="Press Enter"
+        />
+      </div>
     </div>
   );
-};
+}
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<Todo />, rootElement);
+```
+
+Complete example is presented in this [sandbox](https://codesandbox.io/s/icestore-hs9fe), feel free to play with it.
+
+## API
+
+### registerStore
+
+Register store config to global store instance. 
+
+* Paramters
+  - namespace {string} unique name of store
+  - bindings {object} object of store config including state and actions
+* Return value
+  - {object} store instance
+
+### useStores
+
+Hooks of using multiple store.
+
+* Paramters
+  - namespaces {array} arrays of store namespace
+* Return value
+  - {array} array of stores' bindings
+
+### useStore
+
+Hooks of using single store.
+
+* Paramters
+  - namespace {string} store namespace
+* Return value
+  - {object} store's bindings
 
-ReactDOM.render(
-  <Todo />,
-  document.body
+## Advanced use
+
+### async status
+
+icestore has built in async status support in async function, it enables user to access to async functions' loading and error status without defining extra state which will make code more consise and clean.
+
+#### API
+
+* `action.loading` - flag of whether action is executing
+  - Type: {boolean}
+  - Default: false
+* `action.error` - flag of whether action has error after executing
+  - Type: {boolean}
+  - Default: false
+* `action.disableLoading` - flag of whether disable action's loading effect, if set to true, views will not rerender when loading status changed
+  - Type: {boolean}
+  - Default: false
+* `store.disableLoading` - flag of whether disable loading effect globally. Note, action's `disableLoading` flag will always takes priority when both are set
```

Comment:
flag checking the loading status at the global level. An action's `disableLoading` flag will always take priority when both values are set.